aten/src/ATen/native/native_functions.yaml

```diff
@@ -2423,6 +2423,18 @@
 - func: deg2rad.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   supports_named_tensor: True
 
+- func: degrees(Tensor self) -> Tensor
```

Comment:
Add a comment that degrees is an alias for rad2deg and move these under the rad2deg entries